src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -1593,4 +1593,14 @@ namespace winrt::TerminalApp::implementation
         _ShowAboutDialog();
         args.Handled(true);
     }
+
+    void TerminalPage::_HandleQuickFix(const IInspectable& /*sender*/,
+                                       const ActionEventArgs& args)
+    {
+        if (const auto& control{ _GetActiveControl() })
+        {
+            control.OpenQuickFixMenu();
+        }
+        args.Handled(true);

```

Comment:
shouldn't we only mark it as handled if there was a menu to open...?

that way, the shortcut can pass through if there isn't a menu.